src/components/date-picker/styles.css

```
@@ -48,11 +49,11 @@ z-range-picker > div > * {
   position: absolute !important;
 
   border-radius: 2px;
-  box-shadow: 0px 2px 4px 0px rgba(66, 69, 72, 0.4);
+  box-shadow: 0 2px 4px 0 rgba(66, 69, 72, 0.4);
```

per questa shadow c'è il token
```suggestion
  box-shadow: var(--shadow-2);
```